### ISSUE TITLE
Following links to internal base classes leads to 404 error

### ISSUE THREAD

**Issue by rolson** [SwiftDocOrg/swift-doc#177]
Doc for internal classes is not generated (good). But the internal ~~base classes and~~ protocol conformances are listed and appear as links. Following these links naturally leads to 404 (bad) as the doc for those types are not generated. 

One proposal is to not make internal types links.

Here is a screen-shot of some ~~inheritances of internal types~~ conformances of internal protocols showing as links:

![image](https://user-images.githubusercontent.com/3718875/92046217-70d5be00-ed3f-11ea-935a-9e14a5c38fe2.png)

**Comment by mattt** [comment]
@rolson My understanding is that public classes can't inherit from non-public superclasses or adopt non-public protocols. Would you mind sharing a minimal example that reproduces this (and also compiles)?

**Comment by rolson** [comment]
You are correct, and I misspoke, sorry. The issue is with internal protocol conformances. 

This compiles and exhibits the issue:

```swift
import Foundation

public class Foo: Bar { }

internal protocol Bar { }
```

**Comment by Lukas-Stuehrk** [comment]
This has been fixed in the latest release ([1.0.0-beta.6](https://github.com/SwiftDocOrg/swift-doc/releases/tag/1.0.0-beta.6)).
